### ISSUE TITLE
TX_NET_NEW

### ISSUE THREAD

**Issue by chris-githu** [chap-uzima/DataWarehouse#11]
_From @chris-githu on May 15, 2018 13:56_

Hi @irenefaith @vogoti @mkaburu @lmwaniki 

Just from a meeting with CDC/HMIS. Are you aware of this tool by CDC to report on `TX_NET_NEW`?

_Copied from original issue: chap-uzima/IQTools#33_

**Comment by chris-githu** [comment]
It's a big document but this is the site level worksheet
[TX_NET_NEW Site Level.xlsx](https://github.com/chap-uzima/IQTools/files/2007725/TX_NET_NEW.Site.Level.xlsx)

FYI @jmbula @dkemunto @thnesh

**Comment by chris-githu** [comment]
_From @jmbula on May 16, 2018 8:1_

@chris-githu 
Thanks Chris for sharing I don't think I have seen this before, let me go through and give  my comments later.

**Comment by chris-githu** [comment]
_From @irenefaith on May 17, 2018 10:5_

@chris-githu 
I am seeing this tool for the first time.
Is it rolled out yet?Are you incorporating it in Iqtools?

**Comment by chris-githu** [comment]
@vogoti 

As discussed, let's anticipate this request at the end of Q3.
To begin, please extract from DATIM PIVOT our TX_CURR from EMR sites as of Q2.

FYI @mkaburu, @thnesh , @lmwaniki @irenefaith

**Comment by chris-githu** [comment]
_From @lmwaniki on June 26, 2018 9:19_



[TX_CUrr.xlsx](https://github.com/chap-uzima/IQTools/files/2136477/TX_CUrr.xlsx)
@chris-githu -TX curr end of Q2..I have included non EMR sites coz we also need to account for their net new

**Comment by chris-githu** [comment]
@lmwaniki 
Thanks for this.
@vogoti @mkaburu @thnesh @irenefaith 

Please see attached comparison with IQCare DBs for Q2
[TX_CURR_Q2.xlsx](https://github.com/chap-uzima/IQTools/files/2136951/TX_CURR_Q2.xlsx)

- Total DATIM = 44,594
- Total IQCare = 44,594 
  - This is very impressive!
  - +1 in Karura and -1 in Maua evens the total out. 
- Line list of the 44,594 clients.
  - This will serve as a starting point for TX_NET_NEW calculations at the end of this quarter. Keep this file in a safe place! Call me for the password.

**Comment by vogoti** [comment]
@Chris...the hour is upon us, the password please

**Comment by lmwaniki** [comment]
[TX curr details.xlsx](https://github.com/chap-uzima/DataWarehouse/files/2214569/TX.curr.details.xlsx)
@chris-githu @vogoti @irenefaith  June TX curr -DATIM  hereby attached.

**Comment by chris-githu** [comment]
@vogoti @lmwaniki 
What's your approach for this?

**Comment by vogoti** [comment]
@chris-githu @lmwaniki I think having TX_CURR linelists for Q2 and Q3 and identifying who was in Q2 but no longer there in Q3 using the linelists would be a start but then get exit status for those missing

**Comment by lmwaniki** [comment]
@chris-githu @vogoti @irenefaith 
I second Victor's suggestion.
Moving forward we also need to think of non-EMR sites

**Comment by irenefaith** [comment]
@vogoti @lmwaniki     I hope by September we shall not have any non-EMR site.
@chris-githu  afew issues have emerged .1. terminations being backdated to even previous quarter  and having tx_curr in current quarter who were not in previous quarter and don't seem to be TI or new on ART. 
This is affecting tx_net_new.
there is also emerging self transfer outs, and these are the majority of the backdated terminations.How do we address this?..

**Comment by irenefaith** [comment]
@chris-githu yes we can plan for a meeting and discuss this. @lmwaniki  take this up.

**Comment by lmwaniki** [comment]
@chris-githu @irenefaith 
CDC deadline is 30th July, thus we need this ready possibly by Friday 27th

**Comment by lmwaniki** [comment]
@chris-githu @mkaburu @vogoti @irenefaith 
Hi Chris,
we also need to account for peads TX Net New.
one more category which we need to add is graduation to adults (15 years)
Kindly also share the queries with the team for facility based analysis

regards

Lawrence

**Comment by chris-githu** [comment]
@lmwaniki 
Do we have an updated template to track peads v. adults?

**Comment by chris-githu** [comment]
@lmwaniki @mkaburu @vogoti 
Please note that the scripts for these assume you have last quarters tx_curr table already.
We can only run this on the server for now. In the future we need to figure out a way the sites archive the reported line lists.

**Comment by chris-githu** [comment]
@lmwaniki 
Ok, so the same template
- filtered for patients <15 and current on art in the previous quarter
- Add graduation as an exit reason for current
- Do we need this for the new section. 
    - Considering that you can start ART in January as a paed and by March you're and adult.

**Comment by lmwaniki** [comment]
yes we add a section- the scenario is possible but the numbers would be very very few...

**Comment by chris-githu** [comment]
[TX_NET_NEW Template.xlsx](https://github.com/chap-uzima/DataWarehouse/files/2248619/TX_NET_NEW.Template.xlsx)

Will this do?
@lmwaniki @vogoti @irenefaith @mkaburu

**Comment by lmwaniki** [comment]
Yap it should serve the purpose...will we need number who graduated to adult as a entry/source for 15+ to even out the numbers (Totals)?

**Comment by chris-githu** [comment]
Good point @lmwaniki  

[TX_NET_NEW Template.xlsx](https://github.com/chap-uzima/DataWarehouse/files/2249241/TX_NET_NEW.Template.xlsx)

**Comment by lmwaniki** [comment]
the template looks OK

**Comment by chris-githu** [comment]
@lmwaniki Please confirm that the attach file represents what was reported in Q3 for TX_NEW
### EMR Sites
Age Group | Total
--| --
0-15 | 50
15+ | 846

[TX_NEW_Q3.xlsx](https://github.com/chap-uzima/DataWarehouse/files/2253051/TX_NEW_Q3.xlsx)

**Comment by lmwaniki** [comment]
100% correct

**Comment by chris-githu** [comment]
@lmwaniki @vogoti @mkaburu 
Please confirm Q3 TX_CURR numbers reported in DATIM
### EMR Sites
Age Group | DATIM Total | EMR Total
-- | -- | --
0-15 | 2,636 | 2,618
15+ | 42,186 | 42,275

[TX_CURR_Q3.xlsx](https://github.com/chap-uzima/DataWarehouse/files/2253328/TX_CURR_Q3.xlsx)

**Comment by mkaburu** [comment]
@chris-githu the data is correct but Tunyai does not have an EMR

**Comment by chris-githu** [comment]
@mkaburu Noted. Thanks!

Please review report sent via email. Let me know if we are good for Q3.

FYI @lmwaniki @irenefaith @vogoti

**Comment by lmwaniki** [comment]
@chris-githu 
Hi Chris,

Gaitho would wish to have characteristics of those who Died/LTFU for CDC presentations based on our TX_Net_New analysis.
Kindly share with me this line list, with additional variables as below:
Patient ID
Facility
DOB
HIV diagnosis date
Gender
Marital Status
Registration date
ART start date
initial regimen
Last regimen
Regimen switch?
Last vl
Last VL date
Last visit date
Exit date
Exit reason

FYI @vogoti @mkaburu @irenefaith
